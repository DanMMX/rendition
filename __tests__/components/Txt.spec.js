import React from 'react';
import renderer from 'react-test-renderer';
import Provider from '../../src/components/Provider'
import Txt from '../../src/components/Txt'

test('Txt renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <Txt>Hello world</Txt>
    </Provider>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
