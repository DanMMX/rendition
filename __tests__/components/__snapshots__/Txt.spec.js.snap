// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Txt renders correctly 1`] = `
<div
  className="sc-bdVaJa jKopLr"
>
  <div
    className="sc-htpNat fGEkbD sc-bwzfXH cjMihe"
  >
    Hello world
  </div>
</div>
`;
